packages/expo-screen-capture/ios/EXScreenCapture/EXScreenCaptureModule.m

```diff
@@ -0,0 +1,72 @@
+// Copyright 2018-present 650 Industries. All rights reserved.
+
+#import <EXScreenCapture/EXScreenCaptureModule.h>
+
+@interface EXScreenCaptureModule ()
+
+@property (nonatomic, weak) UMModuleRegistry *moduleRegistry;
+@property (nonatomic, strong) UIView *blockView;
+
+@end
+
+@implementation EXScreenCaptureModule
+
+UM_EXPORT_MODULE(ExpoScreenCapture);
+
+- (id)init {
+  if (self = [super init]) {
+      self.blockView = [[UIView alloc] initWithFrame:CGRectMake(0, 0, [[UIScreen mainScreen] bounds].size.height, [[UIScreen mainScreen] bounds].size.height)];
```

Comment:
Should be just able to copy the entire CGRect since the origin of `bounds` is (0, 0):
```suggestion
      self.blockView = [[UIView alloc] initWithFrame:UIScreen.mainScreen.bounds];
```

Reply:
We need to set both the bounding width and height to the greater of the original width or height so that if the user goes from portrait to landscape or landscape to portrait, the entire screen remains covered in either situation

rewriting this now to something like:
```
if (width > height) CGRectMake(0,0,width,width)
else CGRectMake(0,0,height,height)
```